$NetBSD$

The encoding pragma is no longer supported as of Perl 5.26.0

--- gwhois.orig	2015-04-17 21:30:06.000000000 +0000
+++ gwhois
@@ -77,7 +77,6 @@
 #
 
 use LWP::Simple;
-use encoding ':locale';
 use Net::LibIDN;
 
 $ENV{'HOME'}='/var/home/whois' unless defined $ENV{'HOME'};
